### DISCUSSION TITLE
I can't config mini.move to work with LazyVim

### DISCUSSION THREAD

**Discussion by lajoskvcs-at-scale** [LazyVim/LazyVim#1402] [Q&A]
Hey everyone!
I installed mini.move and it doesn't want to work somehow..
I mentioned that the base keymap has A+j and A+k, but I thought mini.move will rebind those keys.
I checked Lazy and it was successfully installed and started.

I'm using iTerm2.

`plugins/mini.lua`
```lua
return {
  {
    "echasnovski/mini.move",
    version = "*",
    opts = {
      mappings = {
        -- Move visual selection in Visual mode. Defaults are Alt (Meta) + hjkl.
        left = "<A-h>",
        right = "<A-l>",
        down = "<A-j>",
        up = "<A-k>",

        -- Move current line in Normal mode
        line_left = "<A-h>",
        line_right = "<A-l>",
        line_down = "<A-j>",
        line_up = "<A-k>",
      },
    },
  },
}
```
Thanks in advance!

**Comment by dpetka2001** [comment]
```lua
return {
  {
    "echasnovski/mini.move",
    version = "*",
    opts = {},
  },
}
```
This worked for me as I installed it for testing purposes. What exactly doesn't work for you?

**Reply by lajoskvcs-at-scale** [reply]
When I press option+h, option+j, option+k or option+l doesn't happen anything

**Reply by dpetka2001** [reply]
You mean the `Alt` key right? Maybe it has something to do with your terminal.

**Reply by dpetka2001** [reply]
Are you using iterm2?

**Reply by dil-mnasir** [reply]
I am facing the same issue and using iterm2? Do I need to change something in iTerm2?

**Reply by dpetka2001** [reply]
Yes, I've read other people having trouble with iTerm2 and `Alt` key. You must configure something in iTerm2 to be able to do that, but unfortunately I don't have MacOS, so I can't help you any further. Your best bet is to google for how to enable `Alt` key in iTerm2 to find a solution. You can also try another terminal emulator such as Wezterm or Kitty to test the behavior.

**Reply by dil-mnasir** [reply]
Yes you are right its with the iTerm2 config, https://stackoverflow.com/questions/196357/making-iterm-to-translate-meta-key-in-the-same-way-as-in-other-oses
